clusterloader2/testing/load/modules/huge-services.yaml

```diff
@@ -18,11 +15,3 @@ steps:
     objectBundle:
     - basename: huge-service
       objectTemplatePath: service.yaml
-{{if $is_deleting}}
-- name: Sleeping after deleting huge services
```

Comment:
As I understand, it is being replaced with DeletionTimeLimited. Could you explain how it exactly works?

Reply:
I've suggested using DeletionTimeLimited. Let's say you have 5 actions and duration is 5 minutes. It will execute 1 action every 1 minute.

Reply:
It is what Marcel said. As per code comment: `TimeLimitedLoad defines a load that spreads operations over given time.`

Reply:
What if we increase huge services size. Would this sleep be needed once again?

Reply:
As far as I understand no, because we would have `n < 5 (iff n = 5000)` operations still spread over 5 minutes

Reply:
Once again - 1000 is just a default value. We will increase it in the future. Target is 10,000

Reply:
We could try to make the time limit a parameter rather than a constant. But I have little idea what it could be... `(pods / 1000) * 1m`? Or somehow reuse `$deletionTime`?

Reply:
@marseel - I'm leaving that to you

Reply:
I would leave it as it is. Once we have 5k or 10k services, I would like to get rid of these multiple deployments.